### ISSUE TITLE
UnicodeEncodeError: 'ascii' codec can't encode character '\xf6' in position 48: ordinal not in range(128)

### ISSUE THREAD

**Issue by kenorb** [mycognitive/google_dl#6]
This command:

```
$ google_dl -x -r -s http://www.example.com/foo/bar/ "foo"
Downloading 'b'%0d%0a\xc3\xb6m_bar.pdf'' from 'b'http://www.example.com/%250d%250a-foo%C3%B6m_bar.pdf'' into . ...
Traceback (most recent call last):
  File "~/google_dl", line 171, in <module>
    print("File '%s' already exists, skipping." % (path))
UnicodeEncodeError: 'ascii' codec can't encode character '\xf6' in position 48: ordinal not in range(128)
```

fails with encoding error.

**Comment by kenorb** [comment]
codemill $25

**Comment by kenorb** [comment]
codemill$25

**Comment by kenorb** [comment]
codemil$25

**Comment by kenorb** [comment]
Fixed in #7
